drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1115,6 +1115,7 @@ function DragAndDropBlock(runtime, element, configuration) {
 
             if (attemptsRemain()) {
                 data.misplaced_items.forEach(function(misplaced_item_id) {
+                    state.last_action_correct = false
```

Comment:
We need to get these PRs merged yesterday... so it's your call as to how this affects the scope. If it's a simple, low-risk cleanup, please get it done now. Otherwise, we can postpone it.